woodwork/schema_column.py

```diff
@@ -10,8 +10,8 @@
 from woodwork.utils import _convert_input_to_set
 
 
-def _get_column_dict(name,
-                     logical_type,
+def _get_column_dict(name=None,
+                     logical_type=None,
```

Comment:
Not a big deal since this is a private function, but we should probably also update the docstring to call out `name` and `logical_type` as optional inputs now.

Reply:
also added a line to the `use_standard_tags` docstring about needing a logical type